### PR TITLE
Push health NewRelic api calls and log

### PR DESCRIPTION
**Pull request by camd**
After looking into New Relic aggregation, I think we MAY be able to get at least part way with just this.  Let's try this first.  If we need to go further later, then we can do that.

### CODE REVIEW

File: treeherder/model/models.py [hunk 1]
```diff
@@ -1394,3 +1394,21 @@ class Meta:
...
             self.text_log_error.id, self.classified_failure.id)
+
+
+class PushActivity(models.Model):
+    """
+    Keep track of events in Treeherder.  This can be used for an event with
+    """
+    id = models.BigAutoField(primary_key=True)
+    push = models.ForeignKey(Push, on_delete=models.CASCADE)
+    time = models.DateTimeField(db_index=True)
```
Comment (klahnakoski) [inline treeherder/model/models.py]: What time?  last updated?  time of event?  What do the other tables use?

File: treeherder/model/models.py [hunk 2]
```diff
@@ -1394,3 +1394,21 @@ class Meta:
...
+    # Whether this entry was done by automated process, or by some user interaction
+    automated = models.BooleanField(default=False)
+    # This could be an enum.  Examples: 'needInvestigation', 'fixedByCommit'
+    type = models.CharField(max_length=20)
+    # Any number to keep track of
+    count = models.SmallIntegerField()
```
Comment (klahnakoski) [inline treeherder/model/models.py]: no need to restrain this number to `Small`, we have plenty of space

File: treeherder/model/models.py [hunk 3]
```diff
@@ -1394,3 +1394,21 @@ class Meta:
...
+    summary = models.CharField(max_length=40)
```
Comment (klahnakoski) [inline treeherder/model/models.py]: longer please

File: treeherder/model/models.py (hunk 1 above)
Comment (camd) [reply treeherder/model/models.py]: I was thinking time of event, yes.

File: treeherder/model/models.py [hunk 4]
```diff
@@ -1394,3 +1394,20 @@ class Meta:
     def __str__(self):
         return "{0} {1}".format(
             self.text_log_error.id, self.classified_failure.id)
+
+
+class PushActivity(models.Model):
+    """
+    Keep track of events in Treeherder.  This can be used for an event with
```
Comment (sarah-clements) [inline treeherder/model/models.py]: ?This can be used for an event with" ??

**Comment by sarah-clements** [comment]
> I want to keep track of `needInvestigation` count over time and in response to doing retriggers to detect intermittents. And I want to log this to New Relic.
> 
I'm curious how you envision this data being used. Would it be a part of Push health or some other view?

**Comment by armenzg** [comment]
I would like to see an example of how it would be used.

**Comment by camd** [comment]
**Notes to self: **
I will look into using Redis or New Relic to be able to group results and accomplish this without a table.
Check in with ionut to see if we could persist data for his needs as well.

**Pull request merged**